packages/migrate/src/__tests__/Baseline.test.ts

```diff
@@ -0,0 +1,115 @@
+import { jestConsoleContext, jestContext } from '@prisma/internals'
+import fs from 'fs-jetpack'
+import prompt from 'prompts'
+
+import { DbPull } from '../commands/DbPull'
+import { MigrateDeploy } from '../commands/MigrateDeploy'
+import { MigrateDev } from '../commands/MigrateDev'
+import { MigrateResolve } from '../commands/MigrateResolve'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+// Disable prompts
+process.env.GITHUB_ACTIONS = '1'
+// Disable generate
+process.env.PRISMA_MIGRATE_SKIP_GENERATE = '1'
+
+// Covered in docs: https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/add-prisma-migrate-to-a-project
+// We have a dev and prod database
+// And we want to baseline the production database with a baseline migration
+// For that, we use `db pull`, `migrate dev`, `migrate resolve` and `migrate deploy` commands
+describe('Baselining', () => {
+  it('SQLite: should succeed', async () => {
+    ctx.fixture('baseline-sqlite')
+    fs.remove('prisma/migrations')
+    fs.copy('prisma/dev.db', 'prisma/prod.db')
+
+    // Start with the dev database
+    process.env.DATABASE_URL = 'file:./dev.db'
+
+    // db pull
+    const dbPull = DbPull.new().parse([])
+    await expect(dbPull).resolves.toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Prisma schema loaded from prisma/schema.prisma
+      Datasource "my_db": SQLite database "dev.db" at "file:./dev.db"
+    `)
+    ctx.mocked['console.info'].mockReset()
+
+    // migrate dev --create-only
+    prompt.inject(['y'])
+    const migrateDevCreateOnly = MigrateDev.new().parse(['--create-only'])
+    await expect(migrateDevCreateOnly).resolves.toMatchInlineSnapshot(`
+      Prisma Migrate created the following migration without applying it 20201231000000_
```

Comment:
The migration name should be sanitized.

Reply:
I don't think this is in scope for this PR.

Reply:
If we want to make a change there, it should go through an issue first. My bias would be to go in the opposite direction and take the user input literally, without tranforming the string in the CLI.